styles/main.css

```diff
@@ -518,3 +529,204 @@ form ul li textarea:focus {
+  padding: 20px 0;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 9px;
+  margin-bottom: 100px;
+}
+
+.snapshoot img {
+  width: 100%;
+}
+
+.text-content {
+  margin-bottom: 40px;
+}
+
+.tonic {
+  padding: 5px 0 5px 20px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.detail {
+  padding: 0 20px 0 20px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.profile {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  padding-left: 20px;
+}
+
+.back-end {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.canopy {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #344563;
+}
+
+.tools {
+  display: flex;
+  gap: 20px;
+  padding-left: 20px;
+}
+
+.tool {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #396df2;
+}
+
+.see-project {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-left: 20px;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background-color: #fff;
+}
+
+/* SECTION DETAIL */
+
+/* MOBILE */
+
+.modal {
+  display: grid;
+  grid-template-rows: auto auto;
+  background-color: #fefefe;
+  margin: 15% auto; /* 15% from the top and centered */
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+  border-radius: 10px;
+}
+
+.modal-div-1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 21px;
+}
+
+.modal-title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+}
+
+.close-modal {
+  font-size: 30px;
+  color: #67798e;
+}
+
+.modal-div-2 {
+  display: flex;
+  gap: 20px;
+  margin: 0 21px;
+}
+
+.modal-img {
+  margin: 0 21px;
+}
+
+.snapshoot1-mob {
+  width: 100%;
+}
+
+.snapshoot1-desk {
+  width: 100%;
+  display: none;
+}
+
+.modal-info {
+  display: grid;
+  grid-template-columns: 60% 40%;
+  margin: 0 21px;
+}
+
+.modal-tools {
+  display: flex;
+  gap: 20px;
+  padding-left: 0;
+  border-bottom: 1px solid #ebecf0;
+  margin-top: 20px;
+}
+
+.modal-btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background-color: #fff;
+  margin: 20px 20px 0 0;
+}
+
+.modal-btn img {
+  margin-left: 5px;
+}
+
+/* MODAL */
+
+.popup {
+  display: none;
+  position: fixed;
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0 0 0); /* Fallback color */
+  background-color: rgb(0 0 0 / 40%); /* Black w/ opacity */
+}
+
+.popup-header {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] The mobile section of your modal is not responsive. Kindly style the section to look like that of Figma's design provided.

### Your Mobile popup
![image](https://user-images.githubusercontent.com/77106962/152648706-8b20fd20-b3d1-46fd-88f2-6f4a9a7af9de.png)

### Figma's Mobile Popup
![image](https://user-images.githubusercontent.com/77106962/152648745-083f0a24-ecc6-4d7d-b285-28655fbea78b.png)

#### Kindly implement this change 🙏

Reply:
Thank you so much. thanks for understanding my situation. You saved me